optionals/compat_rhs_usf3/CfgGlasses.hpp

```diff
@@ -0,0 +1,44 @@
+
+class CfgGlasses {
+    class G_Combat;
+    class rhs_googles_black: G_Combat {
+        ACE_Color[] = {0,0,0};
+        ACE_TintAmount = 16.0;
+        ACE_Overlay = "";
+        ACE_OverlayDirt = "A3\Ui_f\data\igui\rsctitles\HealthTextures\dust_upper_ca.paa";
+        ACE_OverlayCracked = QPATHTOF(textures\HUD\Cracked.paa);
+        ACE_Resistance = 1;
+        ACE_Protection = 0;
+        ACE_DustPath = QPATHTOF(textures\fx\dust\%1.paa);
+    };
+    class rhs_googles_clear: G_Combat {
+        ACE_Color[] = {0,0,0};
+        ACE_TintAmount = 0;
+        ACE_Overlay = "";
+        ACE_OverlayDirt = "A3\Ui_f\data\igui\rsctitles\HealthTextures\dust_upper_ca.paa";
+        ACE_OverlayCracked = QPATHTOF(textures\HUD\Cracked.paa);
+        ACE_Resistance = 1;
+        ACE_Protection = 0;
+        ACE_DustPath = QPATHTOF(textures\fx\dust\%1.paa);
+    };
+    class rhs_googles_orange: G_Combat {
+        ACE_Color[] = {1,0.5,0};
+        ACE_TintAmount = 8.0;
+        ACE_Overlay = "";
+        ACE_OverlayDirt = "A3\Ui_f\data\igui\rsctitles\HealthTextures\dust_upper_ca.paa";
+        ACE_OverlayCracked = QPATHTOF(textures\HUD\Cracked.paa);
+        ACE_Resistance = 1;
+        ACE_Protection = 0;
+        ACE_DustPath = QPATHTOF(textures\fx\dust\%1.paa);
+    };
+    class rhs_googles_yellow: G_Combat {
+        ACE_Color[] = {0,0,-1.5};
+        ACE_TintAmount = 8.0;
+        ACE_Overlay = "";
+        ACE_OverlayDirt = "A3\Ui_f\data\igui\rsctitles\HealthTextures\dust_upper_ca.paa";
+        ACE_OverlayCracked = QPATHTOF(textures\HUD\Cracked.paa);
+        ACE_Resistance = 1;
+        ACE_Protection = 0;
+        ACE_DustPath = QPATHTOF(textures\fx\dust\%1.paa);
```

Comment:
Pretty sure you mean `QPATHTOEF(goggles,textures\fx\dust\%1.paa);`
There are no .paa files in this compatibility component.